src/util.cpp

```diff
@@ -669,19 +669,6 @@ void CLanguageComboBox::OnLanguageActivated ( int iLanguageIdx )
     }
```

Comment:
This appears to be unchanged or am I missing something?  If it is unchanged, please revert.

Reply:
Oh, is it inside a `#ifndef HEADLESS`?

Reply:
Yes, I initially wondered the same, but it's because the function is now also required when building headless.